src/Microsoft.Identity.Web/Microsoft.Identity.Web.csproj

```diff
@@ -94,7 +94,7 @@
     <PackageReference Include="Microsoft.AspNetCore.Authentication.OpenIdConnect" Version="3.1.8" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == 'net462' ">
+  <ItemGroup>
     <PackageReference Include="System.Text.Encodings.Web" Version="4.7.2" />
```

Comment:
This probably shouldn't be included in .net 5 or .net standard, right?

Reply:
this is for a tsa issue. we got the same issue w/all versions.